README.md

```diff
@@ -1,9 +1,14 @@
 # ecAudit - Ericsson Cassandra Audit
 
 [![Build Status](https://travis-ci.org/Ericsson/ecaudit.svg?branch=release/c3.0)](https://travis-ci.org/Ericsson/ecaudit)
+[![Maven Central](https://img.shields.io/maven-central/v/com.ericsson.bss.cassandra.ecaudit/ecaudit_c3.0.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22com.ericsson.bss.cassandra.ecaudit%22%20AND%20a:%22ecaudit_c3.0%22)
```

Comment:
Do you know the reason why the version turns up orange? Orange-yellow-red to me indicates that something is wrong?

Reply:
Agree, a bit disturbing. Could be because the major version is still zero, but just guessing.

In any case, I believe it is configurable. I'll take a look at it.